src/yb/tools/yb-admin_client.h

```diff
@@ -250,9 +252,11 @@ class ClusterAdminClient {
       const PeerId& new_leader_uuid,
       std::unique_ptr<consensus::ConsensusServiceProxy>* leader_proxy);
 
+
   CHECKED_STATUS StartElection(const std::string& tablet_id);
 
-  CHECKED_STATUS MasterLeaderStepDown(const std::string& leader_uuid);
+  CHECKED_STATUS MasterLeaderStepDown(const std::string& leader_uuid,
+      const std::string& dest_uuid);
```

Comment:
I think `new_leader_uuid` argument name is preferable to `dest_uuid`. Also it is reasonable for add default value `std::string()` for it.